tests/setup_module_utils.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ ! -d $1 -o ! -d $2 ] ; then
+    exit 0
+fi
+
+# we need absolute path for $2
+absmoddir=$( cd $2 ; pwd )
+
+for item in $absmoddir/* ; do
+    case $item in
+        *__pycache__) continue;;
+        *.pyc) continue;;
+    esac
+    bnitem=$( basename $item )
+    if [ ! -e $1/$bnitem ] ; then
+        ln -s $item $1/$bnitem
```

Comment:
This works ok for the CI that will always create a fresh tox environment but on a developer system, the tox directory is only re-recreated when needed. Therefore by only adding links it might happen that old links are not cleaned up or cause problems when switching between branches. What do you think?

Reply:
I improved the logic - it will first look in $1 and remove any symlinks to the $2 module_utils - it will then symlink the $2 module_utils under $1